missioncontrol/base/models.py

```diff
@@ -23,14 +23,42 @@ class Meta:
         db_table = 'channel'
 
 
+class Experiment(models.Model):
+    '''
+    Represents an experiment
+    '''
+    name = models.CharField(max_length=100, unique=True)
```

Comment:
I believe experiment names are `max_length=255` in normandy.

Reply:
Good catch! Will fix